doc/Deserializer.md

```diff
@@ -27,4 +29,7 @@ When run with the binary module from our demonstration, it will output:
 (string "code")
 ```
 
-You can view the deserializer code in [Base9/b9/src/deserialize.cpp](https://github.com/b9org/b9/blob/master/b9/src/deserialize.cpp) and [Base9/b9/include/b9/deserialize.hpp](https://github.com/b9org/b9/blob/master/b9/include/b9/deserialize.hpp). 
+You can view the deserializer code in [Base9/b9/src/deserialize.cpp] and [Base9/b9/include/b9/deserialize.hpp]. 
+
+[Base9/b9/src/deserialize.cpp]: https://github.com/b9org/b9/blob/master/b9/src/deserialize.cpp
+[Base9/b9/include/b9/deserialize.hpp]: https://github.com/b9org/b9/blob/master/b9/include/b9/deserialize.hpp
```

Comment:
I would leave out the leading `Base9` in the file path. The link targets look good though.